_topic_map.yml

```diff
@@ -642,7 +645,9 @@ Topics:
   File: idling_applications
 - Name: Analyzing Cluster Capacity
   File: cluster_capacity
-  Distros: openshift-origin,openshift-enterprise
+  Distros: openshift-origin
```

Comment:
@dulek All of your edits look great. I just have a question about line 648. This edit makes the Analyzing Cluster Capacity topic appear in Origin only, when it previously appeared in Origin and Enterprise docs. I think you meant to add a line after File :kuryr to define the distro for the Kuryr SDN Administration. So, I think this should be:

Name: Kuryr SDN Administration
File: kuryr
Distros: openshift-origin 

Please confirm, then we'll be ready to go. Thanks again!

Reply:
@ahardin-rh : My bad, thanks for noticing. It should be fine now.